modules/enet/networked_multiplayer_enet.cpp

```diff
@@ -496,7 +510,7 @@ uint32_t NetworkedMultiplayerENet::_gen_unique_id() const {
 					(uint32_t)OS::get_singleton()->get_unique_ID().hash64(), hash );
 		*/
 		hash = hash_djb2_one_32(
-				(uint32_t)((uint64_t)this), hash); //rely on aslr heap
+				(uint32_t)((uint64_t) this), hash); //rely on aslr heap
```

Comment:
mmmhhh... funny thing, clang-format was the one who changed it in the first place... @akien-mga 
```
Running hook: pre-commit-clang-format

The following differences were found between the code to commit and the clang-format rules:

--- a/modules/enet/networked_multiplayer_enet.cpp	2017-03-24 02:26:21.995201354 +0100
+++ b/modules/enet/networked_multiplayer_enet.cpp	2017-03-24 02:26:26.983158566 +0100
@@ -517,7 +517,7 @@
 					(uint32_t)OS::get_singleton()->get_unique_ID().hash64(), hash );
 		*/
 		hash = hash_djb2_one_32(
-				(uint32_t)((uint64_t)this), hash); //rely on aslr heap
+				(uint32_t)((uint64_t) this), hash); //rely on aslr heap
 		hash = hash_djb2_one_32(
 				(uint32_t)((uint64_t)&hash), hash); //rely on aslr stack
 

Do you want to apply that patch (Y - Apply, N - Do not apply, S - Apply and stage files)? [Y/N/S] 
```